pkg/aws/aws.go

```diff
@@ -320,7 +320,7 @@ func (c Client) create(name string) (*ec2.Reservation, error) {
 				Tags: []*ec2.Tag{
 					{
 						Key:   aws.String("Name"),
-						Value: aws.String(name + "-distros-qa"),
+						Value: aws.String("distros-qa-" + name),
```

Comment:
Do we want to append before or after?

Reply:
Before in my opinion as the resources are typically read left to right in the account. We want any of our assets created to be quickly identified and left alone.

Reply:
ok returned